Cython/Compiler/Main.py

```diff
@@ -253,8 +251,13 @@ def find_include_file(self, filename, pos):
 
     def search_include_directories(self, qualified_name, suffix, pos,
                                    include=False, sys_path=False):
-        return search_include_directories(
-            tuple(self.include_directories), qualified_name, suffix, pos, include, sys_path)
+        if sys_path:
+            include_dirs = self.include_directories + tuple(sys.path)
+        else:
+            include_dirs = self.include_directories
```

Comment:
It might be cleaner to write this as
```
include_dirs = self.include_directories
if sys_path:
    include_dirs = include_dirs + tuple(sys.path)
```

Reply:
adopted